WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -362,12 +391,20 @@ public static boolean hasPendingOrInProgressMediaUploadsForPost(PostModel postMo
         return postModel != null && MediaUploadHandler.hasPendingOrInProgressMediaUploadsForPost(postModel);
     }
 
+    public static boolean hasAnyFailedMediaUploadsForPost(PostModel postModel) {
```

Comment:
Could can drop the `Any` from the method name so it's more consistent with the others?